package.json

```diff
@@ -1,80 +1,84 @@
-    "format": "prettier --plugin-search-dir . --write .",
-    "storybook": "storybook dev -p 6006",
-    "build-storybook": "storybook build"
-  },
-  "exports": {
-    ".": {
-      "types": "./dist/index.d.ts",
-      "svelte": "./dist/index.js"
+    "name": "@nostr-dev-kit/ndk-svelte-components",
+    "version": "0.8.8-2",
+    "scripts": {
+        "dev": "vite dev",
+        "build": "vite build && npm run package",
+        "preview": "vite preview",
+        "package": "svelte-kit sync && svelte-package && publint",
+        "prepublishOnly": "npm run package",
+        "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
+        "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
+        "lint": "prettier --plugin-search-dir . --check . && eslint .",
+        "format": "prettier --plugin-search-dir . --write .",
+        "storybook": "storybook dev -p 6006",
+        "build-storybook": "storybook build"
+    },
+    "exports": {
+        ".": {
+            "types": "./dist/index.d.ts",
+            "svelte": "./dist/index.js"
+        }
+    },
+    "files": [
+        "dist",
+        "!dist/**/*.test.*",
+        "!dist/**/*.spec.*"
+    ],
+    "peerDependencies": {
+        "svelte": "^3.54.0"
+    },
+    "devDependencies": {
+        "@storybook/addon-essentials": "^7.2.0",
+        "@storybook/addon-interactions": "^7.2.0",
+        "@storybook/addon-links": "^7.2.0",
+        "@storybook/blocks": "^7.2.0",
+        "@storybook/manager-api": "^7.2.0",
+        "@storybook/svelte": "^7.2.0",
+        "@storybook/sveltekit": "^7.2.0",
+        "@storybook/testing-library": "^0.2.0",
+        "@storybook/theming": "^7.2.0",
+        "@sveltejs/adapter-auto": "^2.1.0",
+        "@sveltejs/kit": "^1.22.4",
+        "@sveltejs/package": "^2.2.0",
+        "@types/ramda": "^0.29.3",
+        "@typescript-eslint/eslint-plugin": "^6.2.1",
+        "@typescript-eslint/parser": "^6.2.1",
+        "autoprefixer": "^10.4.14",
+        "eslint": "^8.46.0",
+        "eslint-config-prettier": "^8.10.0",
+        "eslint-plugin-storybook": "^0.6.13",
+        "eslint-plugin-svelte": "^2.32.4",
+        "mdsvex": "^0.11.0",
+        "postcss": "^8.4.27",
+        "prettier": "^3.0.1",
+        "prettier-plugin-svelte": "^3.0.3",
+        "publint": "^0.2.0",
+        "react": "^18.2.0",
+        "react-dom": "^18.2.0",
+        "storybook": "^7.2.3",
+        "svelte": "^4.1.2",
+        "svelte-check": "^3.4.6",
+        "tailwindcss": "^3.3.3",
+        "tslib": "^2.6.1",
+        "typescript": "^5.1.6",
+        "vite": "^4.4.8"
+    },
+    "svelte": "./dist/index.js",
+    "types": "./dist/index.d.ts",
+    "type": "module",
+    "dependencies": {
+        "@nostr-dev-kit/ndk": "^0.8.7",
+        "classnames": "^2.3.2",
+        "hurdak": "github:ConsignCloud/hurdak",
+        "isomorphic-dompurify": "^1.8.0",
+        "marked": "^7.0.1",
```

Comment:
This is all formatting changes. The only additions are `isomorphic-dompurify`, `marked`,  `marked-gfm-heading-id`, and `marked-mangle`